### PR TITLE
Add command to copy to system clipboard

### PR DESCRIPTION
**Pull request by tomi2108**

### CODE REVIEW

File: lua/gitportal/init.lua [hunk 1]
```diff
@@ -33,4 +33,16 @@ function M.open_file_in_neovim()
     end
 end
 
+function M.copy_link_to_clipboard()
+    local git_url = git_utils.get_git_url_for_current_file()
+    if git_url ~= nil then
+        if string.find(git_url, "http", 0, true) == nil then
+            cli.log_error("Malformed link detected!")
+        end
+        vim.fn.setreg("+", git_url)
```
[the remark this answers is not in the document]
Comment (tomi2108) [reply lua/gitportal/init.lua:42]: Done 😎

File: lua/gitportal/cli.lua [hunk 2]
```diff
@@ -4,6 +4,10 @@ function M.log_error(message)
     vim.notify(message, vim.log.levels.ERROR)
 end
 
+function M.log_info(message)
+    vim.notify(message, vim.log.levels.INFO)
+end
```
Comment (trevorhauter) [inline lua/gitportal/cli.lua:9]: Good add!

**Review by trevorhauter — APPROVED** [review]
Looks great! Thanks for taking the time to add this. I just took a break from studying and this was a nice find. Works in visual mode too!

**Comment by trevorhauter** [comment]
Oops, should have ran the CI first. It looks like you'll need to ensure your contribution adheres to the StyLua formatting for this project

**Comment by trevorhauter** [comment]
@tomi2108 Woohoo! Let's get this merged

**Comment by tomi2108** [comment]
> Looks great! Thanks for taking the time to add this. I just took a break from studying and this was a nice find. Works in visual mode too!

Thanks! Let me know if anything else is needed, will probably keep an eye on open issues and help with what I can.

**Pull request merged**